src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/commonmodels/Address.java

```diff
@@ -48,6 +50,18 @@ public class Address {
     @JsonProperty("country")
     private String country;
 
+    private static String normalise(String value) {
```

Comment:
Doesn't StringUtils.normalizeSpace return null if null is passed? Not sure of the point of this method.

Reply:
You're right in that it will return null if the value is null so I've gotten rid of the null check 

However, we still need a way to ensure that it is an empty string after it has gone through `StringUtils.normalizeSpace()` then it is equal to null 

Because of that, I'm going to keep this method but without the null check